Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "near-ledger"
-version = "0.6.1"
+version = "0.7.0"
```

Comment:
Do not tick the version, release-plz will take care of it. Just use `chore!:` prefix in your PR description to indicate that there is a breaking change